app/styles/components/_map-tools.scss

```diff
@@ -11,21 +11,45 @@
 	.leaflet-control-map-expand {
 		cursor: pointer;
 	}
+
+    .leaflet-control.leaflet-draw {
```

Comment:
Don't know if it would work for this to place things with [Leaflet control positions](http://leafletjs.com/reference.html#control-'topleft'), but that might be cleaner than setting CSS directly.